### PR TITLE
Update JavaBotBlockAPI info

### PR DESCRIPTION
**Pull request by Andre601**

### CODE REVIEW

File: db/seeds/data/libraries.yml [hunk 1]
```diff
@@ -86,7 +86,7 @@
 
     BotBlockAPI api = new BotBlockAPI.Builder()
         .addAuthToken("bots.ondiscord.xyz", "dsag38_auth_token_fda6gs") // Add a supported site and its token
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:88]: Does this not get an enum?

File: db/seeds/data/libraries.yml [hunk 2]
```diff
@@ -76,7 +76,7 @@
     Andre601. It allows posting your guild counts through the [{{site_name}}
     API](/api/docs) both manually and automatically. You can provide both a JDA
     instance or ShardManager instance, but this is optional. You can also
-    retrieve bot information.
+    retrieve bot and bot list information.
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:79]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 3]
```diff
@@ -86,7 +86,7 @@
 
     BotBlockAPI api = new BotBlockAPI.Builder()
         .addAuthToken("bots.ondiscord.xyz", "dsag38_auth_token_fda6gs") // Add a supported site and its token
-        .addAuthToken("botlist.space", "qos56a_auth_token_gfd8g6")
+        .addAuthToken(Site.BOTLIST_SPACE, "qos56a_auth_token_gfd8g6")
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:89]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 4]
```diff
@@ -99,19 +99,24 @@
     post.startAutoPosting(jda, api);
 
 
-    // Create an instance of GetAction
+    // Create an instance of GetBotAction for GETting bot info
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:102]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 5]
```diff
@@ -99,19 +99,24 @@
...
 
-    GetAction get = new GetAction();
+    GetBotAction getBot = new GetBotAction();
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:104]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 6]
```diff
@@ -99,19 +99,24 @@
...
 
 
-    // Get all supported bot lists
+    // Get complete Bot information.
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:107]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 7]
```diff
@@ -99,19 +99,24 @@
...
 
-    get.getBotLists("myBotId"); // Used for caching
+    getBot.getBotInfo("myBotId");
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:109]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 8]
```diff
@@ -99,19 +99,24 @@
...
 
 
-    // Get information about a bot
+    // Create an instance of GetListAction for GETting list info
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:112]: Added to prod ✅

File: db/seeds/data/libraries.yml [hunk 9]
```diff
@@ -99,19 +99,24 @@
...
 
-    get.getBotInfo("myBotId");
+    GetListAction getList = new GetListAction();
+
+
+    // Get all available lists.
+
+    getList.getLists("randomId");
```
Comment (MattIPv4) [inline db/seeds/data/libraries.yml:119]: Added to prod ✅

File: db/seeds/data/libraries.yml (hunk 1 above)
Comment (Andre601) [reply db/seeds/data/libraries.yml:88]: I wanted to show that it would work with String and an enum respectively...

Comment (MattIPv4) [reply db/seeds/data/libraries.yml:88]: Ah cool.

**Comment by MattIPv4** [comment]
This is all in prod now. I'll do a full seed data regeneration later today or over the weekend. Ty!

**Pull request closed without merging**